helm/fuel-block-committer/templates/deployment.yaml

```diff
@@ -49,7 +49,7 @@ spec:
             httpGet:
               path: /health
               port: http
-            initialDelaySeconds: 10
+            initialDelaySeconds: 60
```

Comment:
Had to bump it because the migration is expected to take around 60s to complete.

Reply:
I ran something similar to this:
```sql
    SELECT COUNT(DISTINCT fb.height)
    FROM fuel_blocks fb
    JOIN bundles b ON fb.height BETWEEN b.start_height AND b.end_height;
```
on all of the databases. Longest execution time was around 50s. 

We have a pruning period set to 8w currently so the amount of entries in the dbs doesn't change.